common/common-api/src/main/java/org/ow2/proactive/scripting/SimpleScript.java

```diff
@@ -82,7 +83,7 @@ public SimpleScript(File file, Serializable[] parameters) throws InvalidScriptEx
      */
     public SimpleScript(URL url, Serializable[] parameters) throws InvalidScriptException {
         // standard scripts (i.e. other than Selection scripts) use url late binding by default
-        super(url, parameters, false);
+        super(url, parameters, !PASharedProperties.LAZY_FETCH_SCRIPT.getValueAsBoolean());
```

Comment:
Should it be worthy to put this eval in an attribute of Script abstract class ? => The property would be evaluate once (and failed at once if the user value isn't valid), and ease its re-usability in other formats class.

Reply:
Makes sense, Ok will change it.